Sources/TemplateKit/ViewRenderer.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/// Renders an Encodable object into a `View`.
+public protocol ViewRenderer: class {
+
+    /// For view renderers that use a cache to optimize view loads, use this variable to toggle whether or not cache should be implemented
+    ///
+    /// Normally, cache is disabled in development so views can be tested w/o recompilation.
+    /// - note: In production, cache is enabled to optimize view serving speed.
+    var shouldCache: Bool { get set }
+
+    /// Renders the template bytes into a view using the supplied `Encodable` object as context.
+    ///
+    /// - parameters:
+    ///     - path: Path to file contianing raw template bytes.
+    ///     - context: `Encodable` item that will be encoded to `TemplateData` and used as template context.
+    /// - returns: `Future` containing the rendered `View`.
+    func render<E>(_ path: String, _ context: E) -> Future<View> where E: Encodable
+}
+
+extension ViewRenderer where Self: TemplateRenderer {
+
+    public var shouldCache: Bool { return true }
```

Comment:
Should cache should be implemented by setting [`astCache`](https://github.com/vapor/template-kit/blob/master/Sources/TemplateKit/Pipeline/TemplateRenderer.swift#L24) to `nil` when false and initializing it to an empty cache when true.